include/crocoddyl/core/numdiff/actuation.hpp

```diff
@@ -80,6 +80,10 @@ class ActuationModelNumDiffTpl : public ActuationModelAbstractTpl<_Scalar> {
    */
   void set_disturbance(const Scalar& disturbance);
 
+  virtual void get_actuated(const boost::shared_ptr<ActuationDataAbstract>& data, Eigen::Ref<VectorXs>& u) {
```

Comment:
This function is declared as virtual. If you expect some classes that won't declare it, then (1) this message should go to the base class and (2) this function declaration should be removed